src/bodypart.h

```diff
@@ -18,6 +18,7 @@ enum body_part : int {
     bp_leg_r,
     bp_foot_l,
     bp_foot_r,
+    bp_blood,
```

Comment:
Blood as a body part is more trouble than it's worth.

You could implement it as a separate value, like oxygen.